e2e/launcher/run-tests.js

```diff
@@ -1,9 +1,8 @@
-import { browsersForBrowserStack, browsersForLocal } from './browsers.js';
+import {browsersForCI, browsersForLocal} from './browsers.js';
 import testcafe from "testcafe";
 
 
 export const Mode = {
-	BrowserStack: 'browserstack',
```

Comment:
Are we no longer using BrowserStack at all?

Reply:
Correct